src/code_80005FD0.c

```diff
@@ -4288,53 +4288,52 @@ void func_80011EC0(s32 arg0, Player *player, s32 arg2, UNUSED u16 arg3) {
     }
 }
 
+#define LENGHT_TRACK_WAYPOINTS(waypoint) for (i = 0; ; i++) { \
```

Comment:
LENGTH_

Reply:
TRACK_WAYPOINTS_LENGTH

or
PATH_LENGTH

This macro may be controversial because it 'hides code' however, I think this is a pretty reasonable macro that does exactly as it says.

CALCULATE_PATH_LENGTH
GET_PATH_LENGTH? <-- maybe this one the best? python, js etc would put the 'length' at the end.

Reply:
it's why I put the macro above two function to see it